dev/run

```diff
@@ -249,9 +249,15 @@ def hack_default_ini(ctx, node, content):
     coffeejs = os.path.join(ctx['rootdir'], "share", "server", "main-coffee.js")
 
     repl = "javascript = %s %s" % (couchjs, mainjs)
+    if os.name == 'nt':
+        # Erlang always wants UNIX-style paths
+        repl = repl.replace("\\", "/")
```

Comment:
Discussion on IRC lead to this approach:
```python
>>> posixpath.join(*("E:\\this\\is\\a\\test".split(os.sep)))
'E:/this/is/a/test'
```
I'll update this PR and repost, then merge (as @iilyak gave me a +1 on IRC)